public/stylesheets/account.css

```diff
@@ -55,3 +55,62 @@ main {
 .account-first-row {
   min-height: 180px;
 }
+.change-profile-form {
+  padding: 10px;
+  padding-right: 50px;
+}
+.profile-form-input-div {
+  display: flex;
+  width: 100%;
+  padding: 5px;
+}
+.profile-form-input {
+  flex-grow: 1;
+  margin-left: 10px;
+  min-width: 30px;
+}
+#bio {
+  box-sizing: border-box;
+}
+.form-section:first-of-type {
+  padding-bottom: 30px;
+}
+.form-section-label {
+  font-size: 20px;
+  border-bottom: 1px solid black;
+  margin-bottom: 10px;
+}
+.form-input-label {
+  font-size: 16px;
+}
+.form-button {
+  float: right;
+  background-color: rgba(1, 172, 1, 0.801);
+  color: white;
+  border: none;
+  border-radius: 5px;
+  padding: 8px 16px;
+  margin-top: 10px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 13px;
+}
+.form-button:hover {
+  background-color: rgba(2, 146, 2, 0.801);
+
+  color: white;
```

Comment:
You shouldn't need to specify the same css attributes as base styles when when styling onhover

Reply:
I've been actually doing that in every stylesheet with bottons. Will change this and the `hover` button stylings in other stylesheets as well.